Puppetfile

```diff
@@ -0,0 +1,3 @@
+mod 'puppetlabs/stdlib', '4.10.0'
+mod 'puppetlabs/apt', '2.2.1'
+mod 'deric/mesos', '0.6.4'
```

Comment:
For `r10k`, yes.

However, I was thinking of switching to [`librarian-puppet`](http://librarian-puppet.com) which has a similar Puppetfile but is much more like Bundler/pip in that you can specify a range of valid versions and it actually does dependency resolution.

Wasn't sure whether to do that here or in a new PR.